### DISCUSSION TITLE
Would like to hide the search field for a specific doc site

### DISCUSSION THREAD

**Discussion by JaniceManwiller** [GitbookIO/community#698] [2. Published content]
I have a [top-level doc site](https://docs.tonic.ai/portal) that is just used to navigate to doc sites for each of our products, and to other non-GitBook information resources. It contains a single GitBook space.

The search for this site is not that useful, because it only finds content within the site, which is just navigation. Nothing to really search for.

Is there a way to hide the search in a specific site?

**Comment by SamyPesse** [comment]
I don't think we'll support such an option. But, we are working on a way to structure multi-product documentation as one single site. We'll ship in a few weeks, and it might help you solve your problem by having one single site with a more powerful structure to represent the multiple products.

With this new feature, the search will cover all of your product and not just the landing page.

Let us know if you want to learn more.
